src/Corrade/Utility/Debug.cpp

```diff
@@ -120,11 +173,12 @@ void Debug::resetColor(Debug& debug) {
     if(debug._flags & InternalFlag::DisableColors) return;
 
     debug._flags &= ~InternalFlag::ColorWritten;
-
+#ifndef CORRADE_TARGET_WINDOWS
     const bool noSpaceBefore = !!(debug._flags & InternalFlag::NoSpaceBeforeNextValue);
     if(!noSpaceBefore) debug << Debug::nospace;
     debug << "\033[0m";
     if(noSpaceBefore) debug << Debug::nospace;
+#endif
```

Comment:
_Resetting_ is not implemented in Win32 API headers. I would have to store the previous state somewhere. I'll have a look into this.